src/app/pages/storage2/modules/devices/components/disk-node/disk-node.component.scss

```diff
@@ -0,0 +1,73 @@
+@import '~assets/styles/mixins/grid';
+
+:host {
+  align-items: center;
+  display: flex;
+  flex: 1;
+  font-weight: 500;
+  min-height: 48px;
+  padding: 0;
+}
+
+.tree-node-grid {
+  @include grid-row();
+  align-items: center;
+
+  .cell {
+    align-items: center;
+    display: inline-flex;
+    min-height: 48px;
+
+    &:first-child {
+      left: 0;
+      position: sticky;
+    }
+  }
+
+  .cell-status {
+    justify-content: center;
+    width: 70%;
+
+    span {
+      padding: 10px 20px;
+    }
+  }
+
+  .cell-name {
+    mat-icon {
+      padding: 5px;
+    }
+  }
+}
+
+// TODO: Fragile (at least with css selectors)
+:host-context(ix-tree-node:hover),
+:host-context(ix-tree-node.selected),
+:host-context(.disk-nested-tree-root-node:hover),
+:host-context(.disk-nested-tree-root-node.selected) {
```

Comment:
I think something got broken here. It used to be that when table got smaller and first item overlapped  other cells, it would have dark background with gradient.
You can compare how it is in datasets.
<img width="408" alt="Снимок экрана 2022-07-11 в 23 25 52" src="https://user-images.githubusercontent.com/1481270/178361448-5234d3c6-09de-4b44-b2fb-230b41ea0545.png">

Reply:
<img width="794" alt="image" src="https://user-images.githubusercontent.com/351613/178469513-ffecfa09-dc0d-4013-8ea4-9bf32fd62b95.png">

Group rows has different styles on mockup

Reply:
Now both when scrolled horizontally and when not, it looks exactly like in Dataset Management page:
![image](https://user-images.githubusercontent.com/20611516/178761012-2feafa52-61b5-40db-b79a-18a231547695.png)

Reply:
![image](https://user-images.githubusercontent.com/351613/178779768-e532ae04-2baa-4378-ae57-9244c711db3b.png)

Sorry for the confusion. I mean `Data VDev` and other group rows not Mirror/Raidz etc